qsirecon/config.py

```diff
@@ -412,43 +412,20 @@ class execution(_Config):
     """Output verbosity."""
     low_mem = None
     """Utilize uncompressed NIfTIs and other tricks to minimize memory allocation."""
-    # md_only_boilerplate = False
-    # """Do not convert boilerplate from MarkDown to LaTex and HTML."""
     notrack = False
     """Do not collect telemetry information for *QSIRecon*."""
     output_dir = None
     """Folder where derivatives will be stored."""
-    output_layout = None
-    """Layout of derivatives within output_dir."""
-    # output_spaces = None
-    # """List of (non)standard spaces designated (with the ``--output-spaces`` flag of
-    # the command line) as spatial references for outputs."""
     reports_only = False
     """Only build the reports, based on the reportlets found in a cached working directory."""
     run_uuid = f"{strftime('%Y%m%d-%H%M%S')}_{uuid4()}"
     """Unique identifier of this particular run."""
-    skip_odf_reports = False
```

Comment:
We'll need to keep this one. This specifically skips creating reports that often cause crashes on hpc nodes without displays